providers/src/airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -256,7 +261,11 @@ def _render_log_id(self, ti: TaskInstance | TaskInstanceKey, try_number: int) ->
         if self.json_format:
             data_interval_start = self._clean_date(data_interval[0])
             data_interval_end = self._clean_date(data_interval[1])
-            execution_date = self._clean_date(dag_run.execution_date)
+            logical_date = (
+                self._clean_date(dag_run.logical_date)
+                if AIRFLOW_V_3_0_PLUS
+                else self._clean_date(dag_run.execution_date)
+            )
```

Comment:
But this one actually doesn’t—`dag_run.logical_date` is available on Airflow 2.

Reply:
Changed it.